server/notifications/services.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict
+
+from django.core import mail
+
+from notifications import models as notify_models
+from django.template import loader
+from django.utils import html
```

Comment:
Imports should be grouped as mentioned in the second [bullet](https://www.python.org/dev/peps/pep-0008/#imports)